tests/regression/test_fdm.py

```diff
@@ -146,6 +147,7 @@ def test_p_independence_hgrad(mesh, variant):
             assert solve_riesz_map(problem, sp) <= expected_it
 
 
+@pytest.mark.skipmumps
```

Comment:
Why can't this use any other direct solver?

Reply:
It's just failing to converge without MUMPS I believe